pandas/core/nanops.py

```diff
@@ -569,7 +664,7 @@ def _get_counts_nanvar(mask, axis, ddof, dtype=float):
             count = np.nan
             d = np.nan
     else:
-        mask2 = count <= ddof
+        mask2 = count <= ddof  # type: np.ndarray
```

Comment:
`mypy` analysis needed an extra hint for the following line. It deduced that `mask2: Union[bool, np.ndarray]` and throws an error as `bool` does not have an `any()` function.